x/subscription/keeper/subscription.go

```diff
@@ -251,3 +252,24 @@ func (k Keeper) CreateSubscription(
 
 	return nil
 }
+
+func (k Keeper) AddProjectToSubscription(ctx sdk.Context, subscriptionOwner string, projectAdmin string, projectName string, enable bool, vrfpk string) error {
+	sub, found := k.GetSubscription(ctx, subscriptionOwner)
+	if !found {
+		details := map[string]string{
+			"subscriptionOwner": subscriptionOwner,
+		}
+		return utils.LavaError(ctx, k.Logger(ctx), "AddProjectToSubscription", details, "can't get subscription")
+	}
+
+	plan, found := k.plansKeeper.GetPlan(ctx, sub.GetPlanIndex())
+	if !found {
+		details := map[string]string{
+			"subscriptionConsumer": projectAdmin,
+			"planIndex":            sub.GetPlanIndex(),
+		}
+		return utils.LavaError(ctx, k.Logger(ctx), "AddProjectToSubscription", details, "can't get plan with subscription")
+	}
+
+	return k.projectsKeeper.CreateProject(ctx, subscriptionOwner, projectName, projectAdmin, enable, sub.GetMonthCuLeft(), plan.GetComputeUnitsPerEpoch(), plan.GetMaxProvidersToPair(), math.MaxUint64, vrfpk)
```

Comment:
I know this is existing code, however -
instead of passing many parameters, either pass "plan", or prepare a "projectstypes.Policy" and pass it. former preferable because subscription doesn't care/know about policies. 
and same for the call to CreateAdminProject()